slatedb/src/compactor.rs

```diff
@@ -158,17 +159,17 @@ pub(crate) enum CompactorMessage {
 /// sorted run. It implements the [`CompactionExecutor`] trait. Currently, the only implementation
 /// is the [`TokioCompactionExecutor`] which runs compaction on a local tokio runtime.
 #[derive(Clone)]
+#[allow(dead_code)]
```

Comment:
I'm guessing this is just used in tests now? (I saw we moved to manually constructing some of this in `db_builder`